<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Тест начался</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../src.qrc">
    <normaloff>:/logo/src/logo.ico</normaloff>:/logo/src/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#f8f8f8;
background-color:#181818;</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_input_test_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_input_user_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="placeholderText">
    <string>Как только вы нажмете на Enter, тест начнется, тут будет появляться вводимый вами текст</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_input_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>771</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_input_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Время</string>
    </property>
    <widget class="QLabel" name="label_input_time_remains">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>секунд</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_input_mistakes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Ошибки</string>
    </property>
    <widget class="QLabel" name="label_input_mistakes_percent">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Всего:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_input_mistakes_total">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Процент:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_input_speed">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>15</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>15</height>
     </size>
    </property>
    <property name="title">
     <string>Скорость</string>
    </property>
    <widget class="QLabel" name="label_input_speed_average_per_sec">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_input_speed_current_per_sec">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>70</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Средняя:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_input_speed_average_per_min">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Текущая:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_input_speed_current_per_min">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>смв/сек</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>смв/мин</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>смв/сек</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>смв/мин</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_input_drop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input_save">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Сохранить результаты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input_quit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Выйти в главное меню</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections/>
</ui>
